gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-agp = "8.2.2"
+agp = "8.1.0"
```

Comment:
Why we need to downgrade to `8.1.0`?

Reply:
I couldn't build & run with agp 8.2.2, and because of this, I downgraded the agp.
With the update to 8.2.2, I fell into gradle build loop :)